maquetado/scss/details.scss

```diff
@@ -26,3 +26,17 @@
   font-size: $font-extra-large;
   margin-right: 10px;
 }
+
+.book-card {
+  .book-image {
+    max-width: 261px;
+    max-height: 368px;
+    padding: 24px;
+  }
+  display: flex;
+  flex-flow: row;
+  justify-content: flex-start;
+  max-width: 900px;
+  max-height: 416px;
+  margin: 5px;
```

Comment:
Why you need this margin?

Reply:
For the screenshot! jajaja. In the next PRs that margin was removed.